packages/next/src/build/webpack/plugins/eval-source-map-dev-tool-plugin.ts

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
can we put a link in comment here as well to indicate where is it forked from?

Reply:
There's a link below where we actually fork: 
```
// Fork of webpack's EvalSourceMapDevToolPlugin with support for adding `ignoreList`.
// https://github.com/webpack/webpack/blob/e237b580e2bda705c5ab39973f786f7c5a7026bc/lib/EvalSourceMapDevToolPlugin.js#L37
```